contrib/bentoml/README.md

```diff
@@ -1,193 +1,209 @@
+fraud_detection_runners = []
+
+for model_name in [
+    "ieee-fraud-detection-0",
+    "ieee-fraud-detection-1",
+    "ieee-fraud-detection-2",
+]:
+    model_ref = bentoml.xgboost.get(model_name)
+    fraud_detection_preprocessors.append(model_ref.custom_objects["preprocessor"])
+    fraud_detection_runners.append(model_ref.to_runner())
+
+svc = bentoml.Service("fraud_detection", runners=fraud_detection_runners)
+
+
+@svc.api(input=PandasDataFrame.from_sample(sample_input), output=JSON())
+async def is_fraud(input_df: pd.DataFrame):
+    input_df = input_df.astype(sample_input.dtypes)
+
+    async def _is_fraud(preprocessor, runner, input_df):
+        input_features = preprocessor.transform(input_df)
+        results = await runner.predict_proba.async_run(input_features)
+        predictions = np.argmax(results, axis=1)  # 0 is not fraud, 1 is fraud
+        return bool(predictions[0])
+
+    # Simultaeously run all models
+    results = await asyncio.gather(
+        *[
+            _is_fraud(p, r, input_df)
+            for p, r in zip(fraud_detection_preprocessors, fraud_detection_runners)
+        ]
+    )
 
-- Serve and test the service.
-    
+    # Return fraud if at least one model returns fraud
+    return any(results)
 ```
-!bentoml serve service.py:svc --reload
 
-2022-11-07T06:50:53+0000 [INFO] [cli] Prometheus metrics for HTTP BentoServer from "service.py:svc" can be accessed at <http://localhost:3000/metrics>.
-2022-11-07T06:50:53+0000 [INFO] [cli] Starting development HTTP BentoServer from "service.py:svc" listening on <http://0.0.0.0:3000> (Press CTRL+C to quit)
+## Deploy to Kubernetes Cluster
 
-```
-    
-![image](https://user-images.githubusercontent.com/861225/212856978-c8a24c4b-bc5b-4706-887e-81f5be914938.png)
+BentoML offers three custom resource definitions (CRDs) in the Kubernetes cluster.
 
-- Build bento
-    
-```
-!bentoml build
+- [BentoRequest](https://docs.bentoml.org/projects/yatai/en/latest/concepts/bentorequest_crd.html) - Describes the metadata needed for building the container image of the Bento, such as the download URL. Created by the user.
+- [Bento](https://docs.bentoml.org/projects/yatai/en/latest/concepts/bento_crd.html) - Describes the metadata for the Bento such as the address of the image and the runners. Created by users or by the `yatai-image-builder` operator for reconsiliating `BentoRequest` resources.
+- [BentoDeployment](https://docs.bentoml.org/projects/yatai/en/latest/concepts/bentodeployment_crd.html) - Describes the metadata of the deployment such as resources and autoscaling behaviors. Reconciled by the `yatai-deployment` operator to create Kubernetes deployments of API Servers and Runners.
 
-Building BentoML service "credit_application:wly5lqc6ncpzwcvj" from build context "."
-Successfully built Bento(tag="credit_application:wly5lqc6ncpzwcvj").
-```
+![image](https://user-images.githubusercontent.com/861225/212857708-f96c9877-bb89-4afa-930a-1d2cb0300520.png)
```

Comment:
Please include the images in the repository itself.

Reply:
Removed this image and moved the detailed example to the BentoML repo. The documentation here will link the full example in BentoML.